packages/clay-card/stories/index.tsx

```diff
@@ -28,13 +30,29 @@ const ClayCheckboxWithState = (props: any) => {
 		<ClayCheckbox
 			checked={value}
 			disabled={false}
+			label={props.label}
 			onChange={() => setValue(val => !val)}
 		>
 			{props.children}
 		</ClayCheckbox>
 	);
 };
 
+const DropDownWithState = (props: any) => {
```

Comment:
No need to declare this as a separate component. We can just include this in the story itself.